_targets/build.project.ia32-generic

```diff
@@ -40,6 +41,7 @@ b_build_target() {
 	b_log "Building sample project for $TARGET"
 
 	# TODO: move to psh install target
+	ln -f "$PREFIX_ROOTFS/bin/"{psh,pshlogin}
```

Comment:
this is unrelated change and not needed anymore, please remove and rebase on top of current master